server/server.go

```diff
@@ -297,11 +283,15 @@ func main() {
     client_api_server.Handle("/client/device", http.HandlerFunc(device))
     client_api_server.Handle("/client/devices", http.HandlerFunc(devices))
 
-
     client_api_server.Handle("/security/scanDevicePort", http.HandlerFunc(scan_port))
 
-    log.Fatal(http.ListenAndServeTLS(":8080", "cert.pem", "key.pem", client_api_server))
+    s := &http.Server{
+        Addr: ":8080",
+        Handler: client_api_server,
+    }
+
+    //log.Fatal(s.ListenAndServeTLS("cert.pem", "key.pem"))
     // Debugging purposes
-    //log.Fatal(http.ListenAndServe(":8080", client_api_server))
+    log.Fatal(s.ListenAndServe(":8080", client_api_server))
```

Comment:
ListenAndServe should be without arguments, since you are already setting them in the http.Server object

From go http docs:
```
s := &http.Server{
	Addr:           ":8080",
	Handler:        myHandler,
	ReadTimeout:    10 * time.Second,
	WriteTimeout:   10 * time.Second,
	MaxHeaderBytes: 1 << 20,
}
log.Fatal(s.ListenAndServe())
```

Reply:
super minor, but 8080 should be a variable up top